projects/1.21/assets/modular-routers/modularrouters/lang/zh_cn.json

```
@@ -0,0 +1,373 @@
+  "modularrouters.guiText.label.playerSect.armor": "盔甲槽",
+  "modularrouters.guiText.label.playerSect.ender": "末影箱物品栏",
+  "modularrouters.guiText.label.playerSect.main": "主物品栏",
+  "modularrouters.guiText.label.playerSect.main_no_hotbar": "主物品栏（不包括快捷栏）",
+  "modularrouters.guiText.label.playerSect.offhand": "副手槽",
+  "modularrouters.guiText.label.regexError": "无效的正则表达式",
+  "modularrouters.guiText.label.selectTag": "选择一条物品标签",
+  "modularrouters.guiText.label.upgrades": "升级",
+  "modularrouters.guiText.label.xpVacuum.eject": "输出",
+  "modularrouters.guiText.label.xpVacuum": "经验类型",
+  "modularrouters.guiText.popup.activator.action": "§a§n行为\n\n执行模块可以模拟下列行为：\n- 使用路由器缓存区的物品右击方块（例如：使用锄，开关拉杆）；\n- 右击附近的实体（例如：使用在路由器缓存区中的桶为牛挤奶）。",
+  "modularrouters.guiText.popup.activator.entity": "§a§n实体模式\n\n执行模块操作时指向周围（3格内）的实体：\n- 最近的实体；\n- 随机选择一个实体；\n- 在该范围的所有实体中轮询选择。",
+  "modularrouters.guiText.popup.activator.look": "§a§n瞄准方向\n\n右击时，路由器可以：\n- 平视：直接瞄准模块方向上的方块，最远距离为4格；\n- 仰视：瞄准模块方向正上方的方块；\n- 俯视：瞄准模块方向正下方的方块。",
+  "modularrouters.guiText.popup.activator.sneak": "§a§n潜行\n\n设置路由器模拟右击时是否应该潜行。",
+  "modularrouters.guiText.popup.augments": "§a§n拓展\n\n这四个栏位用来放置可合成的模块拓展。用于更改或增强模块的功能。 \n\n参阅模块拓展物品提示来了解每个拓展的作用。 \n\n注意，只有手持单个模块才能放置/取出拓展，这是为了避免拓展被复制。",
+  "modularrouters.guiText.popup.breaker.matchType": "§a§n匹配类型\n\n- 匹配掉落物：当破坏方块时，过滤器会匹配该方块掉落的物品（注意，精准采集会影响掉落）\n- 匹配方块：过滤器匹配方块本身，例如过滤器中标记有石头，则会匹配石头方块，不会受精准采集影响（即使掉落物为圆石）",
+  "modularrouters.guiText.popup.detector.signalLevel": "§a§n红石信号等级\n\n在这里你可以指定一个红石信号强度，一个0到15之间（包含0和15）的整数。当路由器缓存区中的物品与过滤器匹配时，路由器将会输出与之相同强度的信号。",
+  "modularrouters.guiText.popup.detector.weakStrong": "§a§n红石信号充能方式\n\n-§o弱充能§r的红石信号只会充能相邻的方块；\n-§o强充能§r 的红石信号除了充能相邻的方块之外，还能使该方块进一步充能其相邻的方块（类似原版的红石中继器）。",
+  "modularrouters.guiText.popup.direction": "§a§n方向控制\n\n可在GUI中该区域指定模块工作的方向。显示的方向基准相对于路由器的正面。默认的方向是“无”，在大多数情况下这意味着放置的模块不会工作，因此你将需要指定一个方向。",
+  "modularrouters.guiText.popup.distributor.direction": "§a§n传输方向\n\n在这里你可以控制路由器向何处发送物品，或从何处接收物品。",
+  "modularrouters.guiText.popup.distributor.strategy": "§a§n分配策略\n\n定义了分配模块分配物品的方式。分别存在四种可以设定的策略：\n\n§e轮询§r - 按顺序依次尝试将物品送入绑定的全部容器。 \n\n§e随机§r - 每次操作将物品随机送入绑定的不同的容器。\n\n§e最近优先§r - 将物品优先送入离路由器最近的绑定的容器 \n\n§e最远优先§r - 将物品优先送入离路由器最远的绑定的容器",
+  "modularrouters.guiText.popup.extruder2.template": "§a§n模板\n\n你可以在这里指定挤出的方块模板；需要注意的是挤出模块放置的并不是真正的方块，而是一种不可合成、不可采集的特殊模板方块。\n\n在模板栏中标记用来挤出的方块；模块会以从左至右、从上至下的方式依次放置设定的方块。你也可以让挤出模块连续放置多个同种物品（单击或右击模板中的物品来调节）。 \n\n注意：你可以放置非方块物品（例如木棍）；虽然这些方块不能被放置到世界中，但是它们可以充当间隔，在挤出的行列中加入空气方块。",
+  "modularrouters.guiText.popup.filter": "§a§n过滤器\n\n在此处标记物品和方块，以精确的控制模块的工作。如果过滤器和缓存区里的物品不匹配，那么这个模块将不会工作，另请参阅下方的过滤器控制按钮。",
+  "modularrouters.guiText.popup.filterControl": "§a§n过滤器控制\n\n这些按钮控制着过滤器如何影响路由器的缓存区。阅读每个按钮的提示以获取更多信息。",
+  "modularrouters.guiText.popup.flinger.pitch": "§a§n抛射俯仰角\n\n这是抛射角度在竖直方向上的偏移量，以度为单位：\n\n- 如果模块朝向是水平的，这表示相对于水平轴（X轴或Z轴）上或下的夹角\n- 如果模块朝向是竖直的，这表示相对于Y轴的夹角。",
+  "modularrouters.guiText.popup.flinger.speed": "§a§n抛射速度\n\n这是路由器将物品扔出去时物品飞出去的速度。 \n\n用一个粗糙的例子来说明，以1.0速度和30°角度扔出去的物品（模块朝向水平方向）会落在2格远的地方。",
```

```suggestion
  "modularrouters.guiText.popup.flinger.speed": "§a§n抛射速度\n\n这是路由器掷出物品的初速度。 \n\n大致来说，以1.0速度和30°角度掷出的物品（模块朝向水平方向）会落在20格远的地方。",
```
另，扔出去/掷出/投掷/抛射，要不要统一说法

---

Fling全改成抛射了，和Drop投掷作区分